jep/223/README.adoc

```diff
@@ -121,7 +121,11 @@ Jenkins configuration settings that will continue to require the "Overall/Admini
 
 * *Manage Users* - Managing users includes the ability to delete any user, so this link will only be shown to users with the `Jenkins.ADMINISTER` permission.
 
+* *About Jenkins - Installed Plugin Depenencies* - Although the `About Jenkins` link is accessible to users with the `Jenkins.MANAGE` permission, we have made the decision to not display the list of installed plugins and their dependencies if the user lacks `Jenkins.ADMINISTER` permissions.  
+This list includes a link for each plugin that takes the user to the plugin management url, and there is no simple way to ensure that all plugins can expose links such as `http://<jenkins_url>/pluginManager/plugin/<plugin>>/thirdPartyLicenses` to a user with only `Jenkins.MANAGE`.
```

Comment:
The better solution IMO would be to not show these links to Jenkins.MANAGE users, or perhaps host the content in a different object they have access to. <10 lines of code I think.

Reply:
@daniel-beck  - not entirely sure i understand what you are proposing 

> The better solution IMO would be to not show these links to Jenkins.MANAGE users

Did you mean we should show the list of installed plugins, but hide the links in that list that the user does not have access to?  That should be easy to do, but i'd also be happy if you have a simple prototype of how we can show them the actual dependency info in a safe way.

Reply:
Without having the links to view license and dependency information that section provides little value other than a list of installed plugins (which is confusing unless we also change the heading shown to a user with only `Jenkins.MANAGE` permission)

Reply:
Change the link in `AboutJenkins/index.jelly` to `${rootURL}/plugin/${p.shortName}/wrapper/thirdPartyLicenses`, and change `thirdPartyLicenses.jelly` to only require `app.MANAGE` permission and it should work.

That bypasses a permission check in `/pluginManager/`, but the object itself checks permissions individually on everything anyway.

Reply:
ok thanks!

Reply:
Addressed in code by https://github.com/aHenryJard/jenkins/pull/7

Will update the jep language to reflect the change